### ISSUE TITLE
Error capture method in Android: NoSuchKeyException: couldn't find key type in dynamic object

### ISSUE THREAD

**Issue by agrass** [lwansbrough/react-native-camera#193]
When I try to use the capture() method I get this error (Same with type, title and description): 

``` 
Exception in native call from JS
unknown:React: com.facebook.react.bridge.NoSuchKeyException: couldn't find key type in dynamic object
unknown:React:  at com.facebook.react.bridge.ReadableNativeMap.getInt(Native Method)
unknown:React:  at com.lwansbrough.RCTCamera.RCTCameraModule.capture(RCTCameraModule.java:154)
unknown:React:  at java.lang.reflect.Method.invoke(Native Method)
```

I get the capture method working, please if anyone can check If the change are ok or not? #192

**Comment by kwngo** [comment]
Getting a similar error running react-native 0.19

**Comment by agrass** [comment]
Yoy can solve it adding ```  type: props.type, ``` into capture method default options in Camera.js or adding in the method capture like this:

```camera.capture({type: 2, title: 'example_title', description: 'example_description'});
```

**Comment by kwngo** [comment]
Thanks, think it worked. I'm getting content://media/external/images/media/22 events in my logs.

**Comment by agrass** [comment]
fixed in PR #192 with default params
